platform/docs/docs/platform/services/data/MeasurementService.md

```diff
@@ -71,6 +71,12 @@ There are seven events that get publish in `MeasurementService`:
 - `jumpToMeasurement(viewportId, id)`: calls the listeners who have
   subscribed to `JUMP_TO_MEASUREMENT`.
 
+- `setLabelConfig(config)`: provide a list of autocompleted labels and its
+  configurations from mode.
+
+- `getLabelConfig(config)`: provide a list of autocompleted labels and its
+configurations from mode.
```

Comment:
Myabe we need the getter, but basically the architecture should be user should be able to define dthe labels via customization service in their mode (it can be any of the modes), and then we internally register it in measurement service

Reply:
As it is used from customization service now, this is removed.